Yosemite/Yosemite/Stores/AvailabilityStore.swift

```diff
@@ -43,7 +43,8 @@ private extension AvailabilityStore {
                               unit: .yearly,
                               earliestDateToInclude: Date(),
                               latestDateToInclude: Date(),
-                              quantity: 1) { result in
+                              quantity: 1,
+                                        forceRefresh: false) { result in
```

Comment:
Super nit: Indentation needs to be updated here.

Reply:
this code will be deleted in https://github.com/woocommerce/woocommerce-ios/pull/7623, so I thought I'd leave it unindented in this PR.